frontend/src/API/MembersAPI.ts

```diff
@@ -69,8 +69,9 @@ export class MembersAPI {
     otherMember: IdolMember,
     category: string
   ): Promise<boolean | undefined> {
+    const encode = (input: string): string => input.replace(/\//g, '*');
     const res = APIWrapper.get(
-      `${backendURL}/member/${otherMember.email}/${Buffer.from(category).toString('base64')}`
+      `${backendURL}/member/${otherMember.email}/${encode(category)}`
```

Comment:
Can you try `encodeURIComponent`? Replacing each `/` with a `*` works but it feels a little hacky to me, since we're just converting it into a random character. What if a category contained a `*` for some reason?